src/controller/ordersControllers.js

```diff
@@ -3,35 +3,35 @@ import orders from '../models/orders';
 
 
 export function postOrder (req, res) {
-    const {error} = validateOrders(req.body);
+	const {error} = validateOrders(req.body);
     
-    if(error) return res.status(400).json({status: 'success', message: error.details[0].message});
+	if(error) return res.status(400).json({status: 'Failed to create order', message: error.details[0].message});
 
-    const order = {
-        id: orders.length + 1,
-        type: req.body.type,
-        name: req.body.name,
-        orderDate: req.body.orderDate,
-        expires: req.body.expires,
-        quantities: req.body.quantities,
-        unitPrice: Math.floor( Math.random() + 200),
-        total: 'N' + req.body.quantities * (Math.floor( Math.random() + 200))
-    };
+	const order = {
+		id: orders.length + 1,
+		type: req.body.type,
+		name: req.body.name,
+		orderDate: req.body.orderDate,
+		expires: req.body.expires,
+		quantities: req.body.quantities,
+		unitPrice: Math.floor( Math.random() + 200),
+		total: 'N' + req.body.quantities * (Math.floor( Math.random() + 200))
+	};
 
-    orders.push(order);
+	orders.push(order);
 
-    res.json(order);
+	res.json({justAdded: order, message: 'order Created'});
```

Comment:
#### Nitpick 
For objects, you should add a space between the curly braces
i.e.
```js
({ justAdded: order, message: 'order Created' })
```
Note the spaces between the contents and the curly braces. 
Also, go through your code and fix every instances of this.

Reply:
Thanks so much, I will fix this

Reply:
I just added a linting rule for this and corrected all appearances on this branch